x-pack/plugins/security_solution/server/lib/app_features/security_kibana_sub_features.ts

```diff
@@ -9,7 +9,6 @@ import { i18n } from '@kbn/i18n';
 import type { SubFeatureConfig } from '@kbn/features-plugin/common';
 import { EXCEPTION_LIST_NAMESPACE_AGNOSTIC } from '@kbn/securitysolution-list-constants';
 import { APP_ID } from '../../../common';
-
```

Comment:
nit: don't remove this line. I thought the linter would complain if there's no empty line after the imports

Reply:
Good catch, thanks! This was leftover from when I had it as a subfeature to the main security category 👍